### PR TITLE
ZCS-1909:Updated the timezones and fixed locator for a failing case

### PR DESCRIPTION
**Pull request by singhjitesh**
- Updated time zones as per the change done in ZCS-1661 as:
       America/Caracas = GMT -04:30 Caracas --> GMT -04:00 Caracas
       America/Santiago = GMT -03:00 Pacific South America --> GMT -04:00 Pacific South America
       Asia/Magadan = GMT +10:00 Magadan --> GMT +11:00 Magadan
       Asia/Novosibirsk = GMT +06:00 Novosibirsk (RTZ 5) --> GMT07:00 Novosibirsk (RTZ 5)
       Europe/Istanbul = GMT +02:00 Istanbul --> GMT +03:00 Istanbul
- Removed the duplicated time zones from the test case.
- Fixed the XPath locator for the case where it was failing when time zone name had "'" in it

### CODE REVIEW

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/preferences/general/VerifyTimezoneView.java [hunk 1]
```diff
@@ -142,114 +133,14 @@ public void VerifyTimezoneView_01() throws HarnessException {
 				"GMT +09:00 Yakutsk (RTZ 8)",
 				"GMT +09:30 Adelaide",
 				"GMT +09:30 Darwin",
-				"GMT +10:00 Magadan",
```
Comment (silentsakky) [inline src/java/com/zimbra/qa/selenium/projects/ajax/tests/preferences/general/VerifyTimezoneView.java:145]: are these timezones removed from webclient?

**Comment by silentsakky** [comment]
we have created TzMsg.properties file for timezone related string translations, don't we need to load this file?

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/preferences/general/VerifyTimezoneView.java (hunk 1 above)
Comment (singhjitesh) [reply src/java/com/zimbra/qa/selenium/projects/ajax/tests/preferences/general/VerifyTimezoneView.java:145]: No, they are present in Web client.

**Comment by singhjitesh** [comment]
No. as of now we don't take timezones information from TzMsg.properties

**Review by miteshsavani — APPROVED** [review]

**Review by jiteshsojitra — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]
LGTM

**Pull request merged**